lisp/ess-trns.el

```diff
@@ -162,7 +162,7 @@ in the region, leaving only the S commands.  Other keybindings are:
   (setq comint-use-prompt-regexp t)
   (make-local-variable 'comint-prompt-regexp)
   (setq comint-prompt-regexp (concat "^" inferior-ess-prompt))
-  (setq inferior-font-lock-keywords 'inferior-R-font-lock-keywords)
+  (setq inferior-ess-font-lock-keywords 'inferior-R-font-lock-keywords)
```

Comment:
@vspinu  - why did you change this recently? Won't that mess up font lock for all non-R inferior modes started from transcript buffers?

Reply:
I didn't change anything substantially here (except of introducing that typo which you fixed). See f51df5dcfc9bf Transcript is in R output only and `inferior-ess-font-lock` is buffer local. Or do you have other issues in mind here?

Reply:
I don't know, but that was the old behavior and I am reasonably sure almost no one is using transcript even for R, let alone stata and sas.